src/Analyzers/CSharp/Tests/RemoveUnusedParametersAndValues/RemoveUnusedParametersTests.cs

```diff
@@ -1647,5 +1647,28 @@ public static ICustomMarshaler GetInstance(string [|s|])
 }
 ");
         }
+
+        [Fact, WorkItem(47174, "https://github.com/dotnet/roslyn/issues/65275")]
+        public async Task TestMethodWithUnusedParameterThrowsExpressionBOdy()
+        {
+            await TestDiagnosticMissingAsync(
+@"public class Class
+{
+    public void Method(int [|x|]) => throw new System.Exception();
```

Comment:
Is there a test that makes sure it still reports if there's more than just a throw?